packages/backend/src/dbt/dbtRpcClientBase.ts

```diff
@@ -212,6 +217,19 @@ export class DbtRpcClientBase implements DbtClient, QueryRunner {
         if (results.request_token) {
             return `${results.request_token}`;
         }
+
+        if (results?.error?.message) {
+            if (
+                results.error.message ===
+                'No server running! Please restart the server.'
```

Comment:
Does this error message come from dbt cloud?

Reply:
Yes 👍